INSTALL.rst

```diff
@@ -470,31 +470,34 @@ Custom Targets
 
 All custom test targets build and run a preconfigured set of tests.
 
-- ``make check [-jN] [ARGS=-jN]``
+- ``cmake --build . -t check [-jN]``
   The default build-and-test target for cvc5, builds and runs all examples,
   all system and unit tests, and regression tests from levels 0 to 2.
 
-- ``make systemtests [-jN] [ARGS=-jN]``
+- ``cmake --build . -t systemtests [-jN]``
   Build and run all system tests.
 
-- ``make units [-jN] [ARGS=-jN]``
+- ``cmake --build . -t units [-jN]``
   Build and run all unit tests.
 
-- ``make regress [-jN] [ARGS=-jN]``
+- ``cmake --build . -t regress [-jN]``
   Build and run regression tests from levels 0 to 2.
 
-- ``make runexamples [-jN] [ARGS=-jN]``
+- ``cmake --build . -t runexamples [-jN]``
   Build and run all examples.
 
-- ``make coverage-test [-jN] [ARGS=-jN]``
+- ``cmake --build . -t coverage-test [-jN]``
   Build and run all tests (system and unit tests, regression tests level 0-4)
   with gcov to determine code coverage.
 
 We use ``ctest`` as test infrastructure, and by default all test targets
 are configured to **run** in parallel with the maximum number of threads
-available on the system. Override with ``ARGS=-jN``.
+available on the system. The default parallelism of the **build** itself
+depends on the underlying build system (e.g., `make` builds using a
+single thread by default, while `ninja` uses all available cores).
 
-Use ``-jN`` for parallel **building** with ``N`` threads.
+Use ``-jN`` to override the number of parallel threads used for both
+building and running the tests.
```

Comment:
I can probably figure out a way and add instructions on how to pass the parallelism flags separately to the build tool and `ctest` if that's important.

Reply:
We believe this clarification is relevant and are happy to keep it.